dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -472,5 +472,9 @@ variables.in.id.not.supported=CWWKG0104W: The ID attribute with value {1} for th
 variables.in.id.not.supported.explanation: Variable values are not allowed in ID attributes. All variables in ID attributes will be treated as literal strings.  
 variables.in.id.not.supported.useraction: Remove the variable from the value of the ID attribute.
 
+error.root.must.be.server=CWWKG0105E: The {0} configuration document must begin with the [{1}] element.
+error.root.must.be.server.explanation=The configuration must be contained in either a server element or a client element. 
```

Comment:
should this message have a colon (:) like the other two or is it supposed to have the (=) ?

Reply:
I'm surprised the build didn't fail for that.. but it looks like the previous message ID in the file has the same issue. I fixed both of them.